hobo.go

```diff
@@ -113,10 +113,18 @@ type localConfig struct {
 }
 
 func newLocalConfigFromFile(fname string) (*localConfig, error) {
+	vmrunPath, err := exec.LookPath("vmrun")
+	if err != nil {
+		return nil, err
+	}
+	vdiskmanagerPath, err := exec.LookPath("vmware-vdiskmanager")
```

Comment:
The problem is that these aren't in the path on OS X, so we need a probe loop to look in the right places so it doesn't break.